lancelot.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "acex88",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/lancelot/10.x/crDroidAndroid-14.0-20231221-lancelot-v10.0-BETA.zip/download",
+			"timestamp": 1703154129,
+			"md5": "7c762ceffb83af19e257533cc5686aba",
+			"sha256": "ebb2295e09bf168e735e1d112e5c72743fa38c487cfdbd36b76a9bfacac71437",
+			"size": 1123150233,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "",
+			"gapps": "",
```

Comment:
no agpps?
maybe set https://nikgapps.com/crdroid-official

Reply:
Ok I will do that